src/Microsoft.Bcl.AsyncInterfaces/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <PackageVersion>1.0.0</PackageVersion>
+    <PackageVersion>1.1.0</PackageVersion>
```

Comment:
@joperezr @terrajobst should we keep these Microsoft.BCL packages in 3.1 since it will be LTS, then delete them from 5.0, or do you want to keep them forever?